serverAddons/PRA3/addons/PRA3_Server/Revive/HitEffects/fn_clientInitHitEffectsUI.sqf

```diff
@@ -0,0 +1,69 @@
+#include "macros.hpp"
```

Comment:
wrong Filename? this render the Information if a Player is damaged and/or need revive or is bleeding